Nightly CI failed on a non-deterministic test case with ticking table and the new Python scope change

```
  ======================================================================
  FAIL [7.211s]: test_ticking_table_scope (tests.test_table.TableTestCase)
  ----------------------------------------------------------------------
  Traceback (most recent call last):
    File "/python/tests/test_table.py", line 724, in test_ticking_table_scope
      self.verify_table_data(rt, list(range(1, 5)))
    File "/python/tests/test_table.py", line 573, in verify_table_data
      self.assertIn(s, t_data)
  AssertionError: 3 not found in array([1, 2, 2, 2], dtype=object)
  
  ----------------------------------------------------------------------
```

--- a/py/server/tests/test_table.py
+++ b/py/server/tests/test_table.py
@@ -719,8 +719,9 @@ class TableTestCase(BaseTestCase):
         self.wait_ticking_table_update(rt, row_count=1, timeout=5)
 
         for i in range(2, 5):
-            x.v = i
-            self.wait_ticking_table_update(rt, row_count=i, timeout=5)
+            with ugp.exclusive_lock():
+                x.v = i
+                self.wait_ticking_table_update(rt, row_count=rt.size + 1, timeout=5)
         self.verify_table_data(rt, list(range(1, 5)))
 
 

